final_test1/main.c

```diff
@@ -0,0 +1,92 @@
+#include <stdio.h>
+#include <string.h>
+#include <stdbool.h>
+#include <stdlib.h>
+
+#define MAX_STRING_LEN 256
+#define MAX_TWO_POWER 4
+#define AMOUNT_OF_DIGIT 3
+
+int lenOctNotationCount(int lenOfBinNotation) {
+    if (lenOfBinNotation % AMOUNT_OF_DIGIT) {
+        return ((int) lenOfBinNotation / AMOUNT_OF_DIGIT) + 1;
+    }
+    return (int) lenOfBinNotation / AMOUNT_OF_DIGIT;
+}
+
+int twoInStartPower(int lenOfBinNotation) {
+    int twoInStartPower = 1;
+    int amountOfNumbersInLastIncompletedTriple = lenOfBinNotation % AMOUNT_OF_DIGIT;
+    if (amountOfNumbersInLastIncompletedTriple) {
+        int startPower = amountOfNumbersInLastIncompletedTriple - 1;
+        for (int i = 0; i < startPower; i++) {
+            twoInStartPower *= 2;
+        }
+        return twoInStartPower;
+    }
+    return MAX_TWO_POWER;
+}
+
+int binaryNotationToOctalNotation(const char *binNotationOfNumber, char *octNotationOfNumber, int lenOfBinNotation) {
```

Comment:
```suggestion
bool binaryNotationToOctalNotation(const char *binNotationOfNumber, char *octNotationOfNumber, int lenOfBinNotation) {
```
Раз возвращаем true или false, то и тип результата - bool.